.github/workflows/_extension_distribution.yml

```
@@ -0,0 +1,965 @@
+        run: |
+          choco install jq -y
+
+      - name: Setup Ccache
+        uses: hendrikmuhs/ccache-action@main
+        continue-on-error: true
+        with:
+          key: ${{ github.job }}-${{ matrix.duckdb_arch }}-${{ inputs.duckdb_version }}
+          save: ${{ inputs.save_cache }}
+
+      - uses: r-lib/actions/setup-r@v2
+        if:  matrix.duckdb_arch == 'windows_amd64_rtools' || matrix.duckdb_arch == 'windows_amd64_mingw'
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)